src/autopas/utils/ArrayUtils.h

```diff
@@ -52,7 +94,43 @@ template <class T>
     strStream << delimiter << *it;
   }
   strStream << surround[1];
+
   return strStream.str();
 }
 
 }  // namespace autopas::utils::ArrayUtils
+
+/**
+ * Stream operator for containers (array and vector types).
+ *
+ * This function actually checks if the given Template parameter satisfies is_container.
+ * Then Generates a string representation of a container which fulfills the Container requirement (provide cbegin and
+ * cend)
+ * @note std::boolalpha is always enabled.
+ * @tparam array or vector of arbitrary types and sizes
+ * @param os string stream
+ * @param container
+ * @return string representation of a container
+ */
+
+template <class Container>
+std::enable_if_t<autopas::utils::ArrayUtils::is_container<Container>::value, std::ostream &> operator<<(
+    std::ostream &os, const Container &container) {
+  const std::string &delimiter = ", ";
+  const std::array<std::string, 2> &surround = {"[", "]"};
+
+  auto it = std::cbegin(container);
+  const auto end = std::cend(container);
+
+  if (it == end) {
+    os << surround[0] + surround[1];
+    return os;
+  }
+  os << std::boolalpha << surround[0] << *it;
```

Comment:
```suggestion
  os  << surround[0] << *it;
```
This is now something that should be set manually on the stream and only if needed